source/classes.tex

```diff
@@ -801,6 +801,7 @@
 \end{codeblock}
 \end{example}
 \begin{note}
+\setlength\emergencystretch{1em}
```

Comment:
I thought \setlength was a LaTeX macro which needs some decorations around the arguments, i.e. `\setlength{\lengthname}{value_in_specified_unit}`.

Reply:
"some decorators" aren't *needed* just for the sake of it -- only if the argument consists of more than one token. In this case it doesn't. But I agree that it'd be nicer to keep the braces even in this degenerate case. I'll change it.

Reply:
Done.